twilio-audio-router/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0.0"
```

Comment:
Should we make this `0.1.0`?

Reply:
Yeah that is fine. We can bump it up to 1.0.0 once we release it as a standalone library.